src/Transports/Mailgun.ts

```diff
@@ -108,7 +108,12 @@ export class MailgunTransport {
 
     Object.keys(tags).forEach((key) => form.append(key, tags[key]))
     Object.keys(headers).forEach((key) => form.append(key, headers[key]))
-    Object.keys(recipients).forEach((key) => form.append(key, recipients[key]))
+
+    Object.keys(recipients).forEach((key) => {
+      if (key !== 'to') form.append('to', recipients[key])
+      form.append(key, recipients[key])
```

Comment:
Shouldn't we remove the 2nd line `form.append(key, recipients[key])`, because I see the mailgun API only accepts the `to` field and any other

Reply:
Yes, I removed the 2nd line `form.append(key, recipients[key])` and opted for `Object.keys(recipients).forEach((key) => form.append('to', recipients[key])` instead.